### PR TITLE
BLOCK-1896-add more examples for group_restriction parameter

### PR DESCRIPTION
**Pull request by Adam-Ultra**
Add more concrete examples for the group_restriction action parameter

### CODE REVIEW

File: docs/contracts/NFT Contract/NFT Actions/setprchsreq.a.md [hunk 1]
```diff
@@ -135,6 +135,51 @@ Refer to [fctrprchs.a](../nft-tables.md#fctrprchs-a)
...
+        * = [NEGATION + GroupA_ID, GroupB_ID]
+        * = [2305843009213693952 + 1, 2]
+        * = [2305843009213693953, 2]
+    - parameter value  ``` "group_restriction": [2305843009213693953, 2] ```
+5. users not belong to GroupA or not GroupB can purchase form this option
+    - logical expression: ~GroupA | ~GroupB 
+    - parameter calculation:
+        * = [NEGATION + GroupA_ID, OR + NEGATION + GroupB_ID]
+        * = [2305843009213693952 + 1, 1152921504606846976 + 2305843009213693952 + 2]
+        * = [2305843009213693953, 3458764513820540932]
```
Comment (keisuke-kanao) [inline docs/contracts/NFT Contract/NFT Actions/setprchsreq.a.md:178]: `3458764513820540932` should be `3458764513820540930`

File: docs/contracts/NFT Contract/NFT Actions/setprchsreq.a.md [hunk 2]
```diff
@@ -192,7 +237,7 @@ await api.transact(
                         },
                         sale_shares: [],
                         maximum_uos_payment: '2.00000000 UOS',
-                        group_restriction: [],
+                        group_restriction: [2305843009213693953, 3458764513820540932],
```
Comment (keisuke-kanao) [inline docs/contracts/NFT Contract/NFT Actions/setprchsreq.a.md:240]: If this is the above case 5, `3458764513820540932` should be `3458764513820540930`

**Review by keisuke-kanao — COMMENTED** [review]
Added comments.

File: docs/contracts/NFT Contract/NFT Actions/setprchsreq.a.md (hunk 1 above)
Comment (Adam-Ultra) [reply docs/contracts/NFT Contract/NFT Actions/setprchsreq.a.md]: updated

File: docs/contracts/NFT Contract/NFT Actions/setprchsreq.a.md (hunk 2 above)
Comment (Adam-Ultra) [reply docs/contracts/NFT Contract/NFT Actions/setprchsreq.a.md]: fixed the calculation. it's my intention to show different parameter combinations, they don't have to be the same. anyway,I changed them to be the same

**Review by keisuke-kanao — APPROVED** [review]

**Pull request merged**